lib/vmdb/loggers/api_logger.rb

```diff
@@ -0,0 +1,39 @@
+require_relative "multicast_logger"
```

Comment:
@Fryguy ah, I see now. This must be pre-autoloading, because the loggers are required [here](https://github.com/ManageIQ/manageiq/blob/master/lib/vmdb/loggers.rb#L92). So, on my machine, api_logger.rb comes before multicast_logger.rb, and inherits from it, so 💥